dbms/src/Storages/DeltaMerge/DeltaMergeStore_Ingest.cpp

```diff
@@ -593,7 +593,7 @@ void DeltaMergeStore::ingestFiles(
         }
         else
         {
-            Remote::DMFileOID oid{.store_id = store_id, .table_id = dm_context->physical_table_id, .file_id = external_file.id};
+            Remote::DMFileOID oid{.store_id = store_id, .keyspace_id = keyspace_id, .table_id = dm_context->physical_table_id, .file_id = external_file.id};
```

Comment:
```suggestion
            Remote::DMFileOID oid{.store_id = store_id, .keyspace_id = dm_context->keyspace_id, .table_id = dm_context->physical_table_id, .file_id = external_file.id};
```
should be the same as line 660